frontend/src/pages/login.tsx

```diff
@@ -1,19 +1,69 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import { AlertCircle } from "lucide-react";
 import Image from "next/image";
-import { ReactElement } from "react";
+import { useRouter } from "next/navigation";
+import { ReactElement, useState } from "react";
 import { FieldValues, SubmitHandler, useForm } from "react-hook-form";
 
+import { GET } from "@/api/requests";
 import Landing from "@/components/Landing";
 import { Textfield } from "@/components/Textfield";
 import { Button } from "@/components/ui/button";
+import { initFirebase } from "@/firebase/firebase";
 import { useWindowSize } from "@/hooks/useWindowSize";
 import { cn } from "@/lib/utils";
 
 export default function Login() {
-  const { register, setValue, handleSubmit } = useForm();
+  const {
+    register,
+    setValue,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
   const _setValue = setValue;
 
+  const [firebaseError, setFirebaseError] = useState("");
+
+  const { auth } = initFirebase();
+  const router = useRouter();
+
+  const login = async (email: string, password: string) => {
+    return await signInWithEmailAndPassword(auth, email, password)
```

Comment:
![Screenshot 2024-04-03 at 9 40 34 AM](https://github.com/TritonSE/PIA-Program-Manager/assets/85322002/89d48e4b-2ddd-4c9b-9d5d-0666505eab76)

When I login in on google chrome I get this error message from google. Not sure what is causing this but it is something to look into.

Reply:
it's just that the sample password we used, abc123, is a pretty insecure password and google doesn't like that